mysql-test/suite/tokudb.rpl/t/rpl_tokudb_row_img_blobs.test

```diff
@@ -0,0 +1,62 @@
+#Want to skip this test from daily Valgrind execution
+--source include/no_valgrind_without_big.inc
+
+#
+# This file contains tests for WL#5096.
+#
+
+-- let $rpl_topology= 1->2->3
+-- source include/rpl_init.inc
+-- source include/have_binlog_format_row.inc
+
+-- connection server_1
+-- source include/have_innodb.inc
```

Comment:
Yeah, it is strange enough that I'll go ahead and fix it, really should be just removing the jump across connections and doing the have_... once at the top. Rather than fully merging it forward to 5.7 (because there were a lot of 5.7 specific re-records and such) I'll just make the similar changes in the 5.7 merge commit.